pangeo-notebook/environment.yml

```diff
@@ -98,6 +100,7 @@ dependencies:
  - xclim
  # - xcube>=0.9.1
  - xesmf
+ - xgboost
```

Comment:
XGBoost is a machine learning algorithm, should this be added to the `ml-notebook` or `pytorch-notebook` instead? Are you using the CPU version of the GPU version of xgboost?

Reply:
Ah ok, if you're just using the CPU version, it should be fine to add xgboost to pangeo-notebook, similar to scikit-learn as you said. Let us know though if you decide to use the GPU version at some point (not sure if the LDEO JupyterHub has GPU clusters?), and we can add it to either `ml-notebook`/`pytorch-notebook` next time.

Reply:
But `cdsapi` is on conda-forge no? See https://anaconda.org/conda-forge/cdsapi. Feel free to remove it though if you prefer to leave it out, and re-run `/condalock` afterwards.

Reply:
So we'll leave `cdsapi` in the docker image then?

Reply:
It makes sense to have cdsapi in the base notebook image. It is a general package for accessing ECMWF data.